src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -4310,6 +4310,10 @@ Copyright (C) Microsoft Corporation. All rights reserved.
            AlwaysCreate="true"
            Condition="'@(ReferencesCopiedInThisBuild)' != ''" />
```

Comment:
Could you instead do this?

```xml
     <Touch Files="@(CopyUpToDateMarker)"
             AlwaysCreate="true"
             Condition="'@(ReferencesCopiedInThisBuild)' != ''">
       <Output TaskParameter="TouchedFiles" ItemName="FileWritesShareable"/>
     </Touch>
```

That is more parallel to the copy task above, and it might be infinitesimally faster to avoid the additional implied item creation below.

Reply:
Yes, that is definitely better. BTW, why `*Shareable`? IIUC, this file should get removed if the project's build outputs are being removed. Please correct me if I'm wrong.

Reply:
You're right, use `FileWrites`, not `-Shareable`. That naming convention never really sunk in for me and I find it confusing every time I have to think about it.